build.gradle

```diff
@@ -1,20 +1,21 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-
+    ext {
+        kotlin_version = '1.5.31'
+    }
```

Comment:
remove this, it isn't required and the kotlin version for our project has been bumped to the latest one